homeassistant/components/moon/sensor.py

```diff
@@ -62,19 +62,19 @@ def device_class(self):
     @property
     def native_value(self):
         """Return the state of the device."""
-        if self._state == 0:
+        if self._state < 0.5 or self._state > 27.5:
             return STATE_NEW_MOON
-        if self._state < 7:
+        elif self._state < 6.5:
```

Comment:
Pylint will complain here since there's a return above. Keep the if on all of these or refactor to just have one return statement.

Reply:
Alright, sure.
I will revert elif's back to if's to keep the PR as small as possible.